src/ui/Form/PasswordField.tsx

```diff
@@ -55,16 +54,16 @@ function PasswordField<FormValues extends FieldValues>(
       </div>
 
       <Form.TextWrapper>
-        {fieldState.invalid && (
-          <Form.ErrorMessage id={`error-${props.id}`}>
+        {fieldState.invalid ? (
```

Comment:
Same as Checkbox. Helper text should be visible.

Reply:
Same as [here](https://github.com/Andrewnt219/rent-near-me/pull/96#discussion_r743710763)